### ISSUE TITLE
I am getting below errors while pulling repo.

### ISSUE THREAD

**Issue by rjbhayani** [rayatus/sapbugtracker#2]
![image](https://user-images.githubusercontent.com/19501487/81187853-3565f580-8fd2-11ea-807b-44f072bc8b4f.png)
![image](https://user-images.githubusercontent.com/19501487/81187930-54648780-8fd2-11ea-96cd-3ffa623253b6.png)

**Comment by rayatus** [comment]
Yes sorry, there are some artifacts that were not correctly ported from Google Code to abapGit and some that I think are not yet supported by abapGit.

I'll try to fix, by the way if you want to have a look at this maybe with the TR's that are saved in "https://code.google.com/archive/p/sapbugtracker/downloads" you could have a try.

let me know if those TR's worked for you.

**Comment by rjbhayani** [comment]
Thank you so much for the same.





[image: photo]
Rahul Bhayani
SAP MDG Consultant

p: +91 9870 687 973 ( India )
e: rjbhayani@yahoo.com
[image: facebook icon] <https://www.facebook.com/rjbhayani> [image: twitter
icon] <https://twitter.com/rjbhayani>  [image: linkedin icon]
<https://www.linkedin.com/in/rjbhayani/> [image: instagram icon]
<https://www.instagram.com/rjbhayani/>
*Information included in this email is reserved to named addressee’s eyes
only. You may not share this message or any of its attachments to anyone.
Please note that as the recipient, it is your responsibility to check the
email for malicious software. *
